css/util/styled_button.css

```diff
@@ -0,0 +1,22 @@
+.styled-button {
+    width: 100%;
+    border: none;
+    border-radius: 5px;
```

Comment:
Do you need this then you actually have no border? 🤔

Reply:
I don't want a border on the button but I want to have rounded corners - hence the border-radius :)